Dockerfile-release.amd64

```diff
@@ -1,5 +1,8 @@
-# TODO: move to k8s.gcr.io/build-image/debian-base:bullseye-v1.y.z when patched
-FROM debian:bullseye-20220328
+FROM --platform=linux/amd64 busybox:1.34.1 as source
+FROM --platform=linux/amd64 gcr.io/distroless/base-debian11
+
+COPY --from=source /bin/sh /bin/sh
+COPY --from=source /bin/mkdir /bin/mkdir
```

Comment:
is there any components using command `/bin/mkdir`?

Reply:
see line 7 and 8 in this file.